windows/scripts/VMware-tools/remove-tools.ps1

```diff
@@ -0,0 +1,34 @@
+
+Write-Host "`n`nRemoving VMware tools from your VM machine`n`n"   -ForegroundColor red -BackgroundColor white
+
+function removevmtools {
+
+try {
```

Comment:
Add proper indenting to the script to make it easier to read & more professional.